study-html/stylesheet.css

```
@@ -0,0 +1,184 @@
+body {
+  margin: 0;
+  font-family: "Hiragino Kaku Gothic ProN";
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  width: 1170px;
+  padding: 0 15px;
+  margin: 0 auto;
+}
+
+.top-wrapper {
+  padding: 180px 0 100px 0;
+  background-image: url(https://prog-8.com/images/html/advanced/top.png);
```

url も文字列などで、ダブルクオーテーションで囲いましょう

```suggestion
  background-image: url("https://prog-8.com/images/html/advanced/top.png");
```